releasenotes/notes/process-agent-fix-json-output-bc58ab39de316bc4.yaml

```diff
@@ -0,0 +1,3 @@
+---
+fixes:
+  - `process-agent check --json` now outputs valid json.
```

Comment:
The `--json` flag is newly introduced in this release, so this note might be a bit misleading/confusing given that it was never "broken".

@maycmlee curious what you think?

Reply:
That's a great point. I was personally going back and forth on whether or not to include this for that exact reason. While the `--json` output is newly added, in fact json output has been broken for many releases, and so I figured it was at least worth mentioning.

I am also curious about what @maycmlee has to say about this.

Reply:
Hi! What about something like, "Adds new `--json` flag. Running `process-agent check --json` outputs valid json."

Reply:
I think that will work. While the `--json` flag was added in https://github.com/DataDog/datadog-agent/pull/11480/files#diff-dd84c63abd7d921936a48fc4f088b6dd01d9cd529e54f9961de44720c3094c80R3, assuming it all gets concatenated together I don't see a problem with it.